hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestHSync.java

```diff
@@ -153,7 +155,7 @@ public class TestHSync {
   private static final BucketLayout BUCKET_LAYOUT = BucketLayout.FILE_SYSTEM_OPTIMIZED;
 
   private static final int CHUNK_SIZE = 4 << 12;
-  private static final int FLUSH_SIZE = 2 * CHUNK_SIZE;
+  private static final int FLUSH_SIZE = 3 * CHUNK_SIZE;
```

Comment:
Just curious is there any particular reason behind this increase in size?

Reply:
Nah, I put it to make the test work when BufferPool can't handle allocation pass capacity.

Reply:
Ah, I see :D